config.toml

```diff
@@ -61,21 +61,35 @@ anchor = "smart"
 # Language configuration
 
 [languages]
-[languages.en]
-title = "EGI Docs"
-description = "EGI documentation"
-languageName ="English"
-# Weight used for sorting.
-weight = 1
+  [languages.en]
+  title = "EGI Docs"
+  description = "EGI documentation"
+  languageName ="English"
+  # Weight used for sorting.
+  weight = 1
+
+[markup]
+  [markup.goldmark]
+    [markup.goldmark.renderer]
+      unsafe = true
+  [markup.highlight]
+      # See a complete list of available styles at https://xyproto.github.io/splash/docs/all.html
+      style = "tango"
+      # Uncomment if you want your chosen highlight style used for code blocks without a specified language
+      # guessSyntax = "true"
 
 # Everything below this are Site Params
 
 [params]
-copyright = "EGI Foundation"
+copyright = "EGI Foundation."
 privacy_policy = "https://www.egi.eu/privacy-policy/"
 
+custom_css = ["css/shortcodes/tabbed-panex.css", "css/tags.css", "css/egi.css"]
+custom_js = ["js/clipboard.min.js", "js/featherlight.min.js", "js/highlight.pack.js", "js/egi.js", "https://buttons.github.io/buttons.js"]
```

Comment:
Can you download and include buttons.js instead of relying on an external an uncontrolled source?

Reply:
Done